saleor/webhook/payloads.py

```diff
@@ -1080,3 +1082,304 @@ def generate_excluded_shipping_methods_for_checkout_payload(
         ],
     }
     return json.dumps(payload, cls=CustomJsonEncoder)
+
+
+def _generate_order_prices_data_with_taxes(
+    order: "Order",
+    manager: PluginsManager,
+    lines: Optional[Iterable[OrderLine]] = None,
+) -> Dict[str, Decimal]:
+    def qp(price: Decimal) -> Decimal:
+        return quantize_price(price, order.currency)
+
+    shipping = order_calculations.order_shipping(order, manager, lines)
+    shipping_tax_rate = order_calculations.order_shipping_tax_rate(
+        order, manager, lines
+    )
+    total = order_calculations.order_total(order, manager, lines)
+    undiscounted_total = order_calculations.order_undiscounted_total(
+        order, manager, lines
+    )
+
+    return {
+        "shipping_price_net_amount": qp(shipping.net.amount),
+        "shipping_price_gross_amount": qp(shipping.gross.amount),
+        "shipping_tax_rate": shipping_tax_rate,
+        "total_net_amount": qp(total.net.amount),
+        "total_gross_amount": qp(total.gross.amount),
+        "undiscounted_total_net_amount": qp(undiscounted_total.net.amount),
+        "undiscounted_total_gross_amount": qp(undiscounted_total.gross.amount),
+    }
+
+
+def _generate_order_prices_data_without_taxes(
+    order: "Order",
+    included_taxes_in_price: bool,
+) -> Dict[str, Decimal]:
+    def quantize_base_price(price: TaxedMoney) -> Decimal:
+        return quantize_price(
+            _get_base_price(price, included_taxes_in_price), order.currency
```

Comment:
I would name it something like `untaxed_price_amount`. Untaxed because we're using a value stored in Saleor DB. `price_amount` because you're returning a decimal value

Reply:
I don't think that's too big of a problem. `quantize_base_price` doesn't add any new logic, it just combines two calls of existing functions to decrease repetition and increase readability.